droidlet/interpreter/craftassist/facing_helper.py

```diff
@@ -26,9 +26,8 @@ def number_from_span(span):
 
 class FacingInterpreter:
     def __call__(self, interpreter, speaker, d):
-        # get these from memory, not player struct!!!!! FIXME!!!
-        current_pitch = interpreter.agent.get_player().look.pitch
-        current_yaw = interpreter.agent.get_player().look.yaw
+        self_mem = interpreter.memory.get_mem_by_id(interpreter.memory.self_memid)
```

Comment:
Do we foresee more of these instances where fetching via memory (instead of directly through agent) is going to result into longer queries ? If yes, might be a good forcing function to refactor memory soon to make these prettier.

Reply:
I am trying to do the minimal changes to get agent out of objects here; I think longer term we need to decide on the api that agent uses to route requests to and results from memory